front/pages/w/[wId]/subscription/index.tsx

```diff
@@ -314,3 +451,91 @@ export default function Subscription({
     </AppLayout>
   );
 }
+
+function SkipFreeTrialDialog({
+  show,
+  onClose,
+  onValidate,
+  workspaceSeats,
+  estimatedMonthlyBilling,
+  isSaving,
+}: {
+  show: boolean;
+  onClose: () => void;
+  onValidate: () => void;
+  workspaceSeats: number;
+  estimatedMonthlyBilling: number;
+  isSaving: boolean;
+}) {
+  return (
+    <Dialog
+      isOpen={show}
+      title={`Skip trial`}
+      onCancel={onClose}
+      validateLabel="Upgrade now"
+      validateVariant="primary"
+      onValidate={() => {
+        onValidate();
+      }}
+      isSaving={isSaving}
+    >
+      <Page.P>
```

Comment:
Not resolved here! 

Also, I would rework the wording -> What is the point of ending the free trial before end of the billing period, is it allowing for more users in the workspace? If this is to prevent a limitation, then I would say it here. 

If yes, I would say something like: 
> Ending your trial period allows you to benefit from xxxxx. Billing will start today for the xx member(s) of your workspace. 

If not, I would still change the wording to something like: 
> Billing will start today for the xx member(s) of your workspace.

Reply:
For the record: there are limits in the free trial, mostly because:
- We don;'t want people to abuse our models API keys.
- We don't want someone to invite 100 people on day one, forget about it, and be charged 29*100 after 14 days.